include/thingset.h

```diff
@@ -1764,6 +1785,50 @@ struct thingset_data_object *thingset_iterate_subsets(struct thingset_context *t
 int thingset_import_data(struct thingset_context *ts, const uint8_t *data, size_t len,
                          uint8_t auth_flags, enum thingset_data_format format);
 
+#ifdef CONFIG_THINGSET_PROGRESSIVE_IMPORT_EXPORT
+/**
+ * Begins the import of data into data objects.
+ *
+ * This function can be used to initialize data objects from previously exported data (using
+ * thingset_export_subsets function) and stored in the EEPROM or other non-volatile memory.
+ *
+ * Unknown data items are silently ignored.
+ *
+ * @param ts Pointer to ThingSet context.
+ * @param data Buffer containing ID/value map that should be written to the data objects
+ * @param len Length of the data in the buffer
+ * @param format Protocol data format to be used (text, binary with IDs or binary with names)
+ */
+int thingset_begin_import_data_progressively(struct thingset_context *ts, const uint8_t *data,
```

Comment:
Would it be possible to combine these 3 functions into one function similar to the `export` part?